### ISSUE TITLE
Map Export Missing Tileset

### ISSUE THREAD

**Issue by mechakaiju** [browserquest/BrowserQuest#164]
Adding new tileset in Tiled Map Editor because the resources added in single file are getting too long (i.e. Photoshop doesn't like to save for web dialog). In order to address the issue I did the steps listed below.
1. Updated tmx file with new tileset and named it tilesheet2.
2. Placed item from tileset on map.
3. Exported map as json.
4. Updated processmap.js  line 85 from the following:

if (tileSetName === "tilesheet") {
to the following:
if (tileSetName === "tilesheet" || tileSetName === "tilesheet2") {

Ran the process the update the client and server direct and the new item is not there. Not sure what I am doing wrong here...any help would be appreciated.

**Comment by Flavius256** [comment]
I'm a bit confused about your issue, have you set the item Type on the Tilemap?

**Comment by mechakaiju** [comment]
Yeah unfortunately it looks like the BrowserQuest engine itself doesn't process multiple tilesheets if they are added into Tiled Map. So while you can update the process map, the engine itself only loads one tilesheet into the array.

**Comment by Flavius256** [comment]
why not add to the current tilesheet?
And it works with me .-.

**Comment by mechakaiju** [comment]
I have...but the length pf the png has gotten too long and photoshop
doesn't like it and won't save as a web document anymore. so I wanted to
look into adding another
